packages/mc-scripts/src/config/optimizations.ts

```diff
@@ -0,0 +1,66 @@
+import chalk from 'chalk';
+
+// Dependencies to be split/grouped into separate chunks.
+// This is useful to reduce the main bundle size and have more
+// dedicated caching strategy for specific chunks.
+const manualChunks = {
+  'commercetools-uikit-icons': ['@commercetools-uikit/icons'],
+  moment: ['moment', 'moment-timezone'],
+  'app-shell': [
+    '@commercetools-frontend/application-shell',
+    '@commercetools-frontend/application-shell-connectors',
+  ],
+};
+
+const removeNonExistantDependencies = (
+  manualChunks: Record<string, string[]>,
+  appDependencies: string[]
+) => {
+  return Object.entries(manualChunks).reduce((acc, [chunkName, vendors]) => {
```

Comment:
Nit: let's use a more meaningful name

```suggestion
  return Object.entries(manualChunks).reduce((chunkGroups, [chunkName, vendors]) => {
```

Reply:
Updated here: f8f062ce27d6b580d54781cd3838f079cf43099d